blog/en/blog/2023/03/17/weekly-report-0312.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Biweekly Report (Feb 27 - Mar 12)
+keywords:
+- Apache APISIX
+- API Gateway
+- Weekly Report
+- Contributor
+description: The cloud-native API gateway Apache APISIX has added functions such as enabling opentelemetry plugin to support https upstream and adding 'range_id' algorithm for 'request-id' plugin.
+tags: [Community]
+image: https://static.apiseven.com/uploads/2023/03/17/pf3ERDxP_%E5%8F%8C%E5%91%A8%E6%8A%A5%E5%B0%81%E9%9D%A2-0317.png
+---
+
+<!--truncate-->
+
+> From 2.27 to 3.12, 29 contributors submitted 74 commits for Apache APISIX. Thank you for your contributions to Apache APISIX.
+
+## Introduction
+
+Apache APISIX grew up as a community from the first day it was open-sourced, and quickly became the most active open-source API gateway project in the world. These achievements are inseparable from the joint efforts of community partners.
+
+"If you want to go fast, go alone. If you want to go far, go together." The Apache APISIX Community Biweekly Report can help community members better grasp the progress of the Apache APISIX community so that everyone can participate in the Apache APISIX community.
+
+We have also sorted out some issues for newcomers to the community to participate in! If you are interested, don't miss it!
```

Comment:
```suggestion
We have also sorted out some issues for newcomers to the community to participate in! If you are interested, don't miss it!
```
this weekly report don't have the part of Good First Issue，so chinese version delete this part